devs/resources.md

```diff
@@ -0,0 +1,78 @@
+# Developer Resources
+
+Here is a list of libraries, examples, SDKs and other resources useful to those developing on the DeSo Blockchain.
+
+## Important
+
+These are unofficial packages and examples provided by the user community and therefore have not been checked for bugs, issues, or security risks.
+
+So use them as examples and Do Your Own Research before running these.
+
+## Opensource Front ends & Extensions
+
+* [Diamond App](https://github.com/diamond-app/frontend)
+* [Bitclout](https://github.com/deso-protocol/frontend)
+* [CloutFeed](https://github.com/CloutFeed/mobileApp)
+* [Plus+ Extension](https://github.com/iPaulPro/BitCloutPlus)
+* [CloutCLI](https://github.com/andrewarrow/cloutcli)
+* [Netlify Hosted Frontend](https://github.com/DeSoDev/BitFlare)
+
+## SDKs / Libraries 
+
+* [Pythton Package](https://pypi.org/project/bitclout/) which supports various api functions: 
+
+    - Get DESO price
+    - Get user info via publickey or username 
+    - Get user profile picture
+    - Get user wallet
+    - Get coin holders
+    - Get list of users blocked by a user
+    - Get posts by a user
+    - Get post information
+    - Get diamond information
+    - Get deleted posts by user
+    - Buy creator coin for a user
+    - Sell creator coin for a user
+    - Create a post
+    - Send a direct message
+
+* [Dart SDK](https://github.com/DeverseSocial/deso_sdk) - Supporting all API endpoints
+
+* [Typescript SDK](https://github.com/bitclouthunt/bitclout-sdk):
+
+    - Get Exchange Rate
+    - Get App State
+    - Get coin holders
+    - Get single user profile by pubkey or username
+    - Get information about 1 or more users
+    - Get a users followers
+    - Get notifications
+    - Get transaction
+    - Submit transaction
+    - Identity service
+
+* [Cloudflare Workers DESO Embed Code](https://github.com/DeSoDev/EmbeDeSo) - Enables DESO urls from showing good embed info on twitter, discord, and in google search.
+
+## Code examples
+
+* Use Identity login:
+
+    - [Javascript](https://github.com/mubashariqbal/login-with-bitclout)
+    - [React](https://github.com/BogdanDidenko/react-bitclout-login) 
+    - [Svelte](https://github.com/mvanhalen/bitclout-identity-window-svelte) 
+    - [Deno](https://github.com/PrismWeb3/deso-web-identity) 
```

Comment:
```suggestion
    - [Javascript](https://github.com/PrismWeb3/deso-web-identity) 
```

Our package actually isn't written in or for Deno; It's build purely for web applications. It does utilizes some things from the  the Deso STD, but all of this is packaged together, so no need to actually use deno in any way.

Reply:
got merged before i could change this so ill add this in update in a bit